Bootcamp.AutomatedBackupService/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bootcamp.Data;
+using Bootcamp.Data.Context;
+using Bootcamp.Data.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Bootcamp.AutomatedBackupService
+{
+    internal class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddDbContext<EngagementDbContext>(options =>
+                    options.UseSqlServer("Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=Bootcamp2;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False;MultipleActiveResultSets=true;")
```

Comment:
fetch this from appsettings

Reply:
fixed in this commit [79104cf](https://github.com/pushkarcopilot/Bootcamp-Training-NV-API/pull/6/commits/79104cf0c6d4881f6a93181b9375ba3b1274019f)